packages/config/src/.eslintrc

```diff
@@ -26,6 +27,7 @@
     "quotes": ["error", "single"],
     "object-curly-spacing": ["error", "always"],
     "indent": ["error", 2],
-    "comma-dangle": ["error", "always-multiline"]
+    "comma-dangle": ["error", "always-multiline"],
```

Comment:
Prettier removes dangling commas from function declarations/calls by default, so `.prettierrc` would need to be updated as well

Reply:
ok, will update that

this rule was not added in this PR though ;)
I only added the last comma in it, to add the next line
that's why it's highlighted
which is a meta-illustration of why it's a good rule diff-wise.
if there was a trailing comma, the line wouldn't be confusingly highlighted

unfortunately trailing commas in JSON are not possible, so it doesn't apply here (although this seems to be JSONC which apparently can tolerate trailing commas)

Reply:
Hah, my bad 😄 that's a good demonstration of the problem indeed